src/auth/project-config.ts

```diff
@@ -97,7 +118,12 @@ export class ProjectConfig {
    */
   public static buildServerRequest(configOptions: UpdateProjectConfigRequest): ProjectConfigClientRequest {
     ProjectConfig.validate(configOptions);
-    return configOptions as ProjectConfigClientRequest;
+    const request = configOptions as any;
+    if (configOptions.multiFactorConfig !== undefined) {
+      request.mfa = MultiFactorAuthConfig.buildServerRequest(configOptions.multiFactorConfig);
+    }
+    delete request.multiFactorConfig;
```

Comment:
I must have missed this, but whey do we have both `multiFactorConfig` and `mfa` in the request type?

Reply:
Because the backend accepts `mfa` but we have `multiFactorConfig` for the field in the SDK to maintain consistency with tenant flow. So this is just a conversion.

Reply:
Reopening since the comment is not here yet. IMO, it is ok to repeat the comment in multiple places.

Reply:
@pragatimodi

Reply:
`link to the docs in the comment` Not sure which doc you are talking about here

Reply:
i meant https://cloud.google.com/identity-platform/docs/reference/rest/v2/projects.tenants#resource:-tenant and https://cloud.google.com/identity-platform/docs/reference/rest/v2/Config

Reply:
Adding to this thread - the reason why SDK is using `multiFactorConfig ` is because this is the approved name by API council.